engine/src/main/java/org/terasology/rendering/opengl/GLSLShader.java

```diff
@@ -309,7 +312,7 @@ private void dumpCode(int type, Set<ShaderProgramFeature> features, String sourc
         }
 
         // Dump all final shader sources to the log directory
-        final String strippedTitle = getUrn().toString().replace(":", "-");
+        final String strippedTitle = COLON.matcher(getUrn().toString()).replaceAll(Matcher.quoteReplacement("-"));
```

Comment:
dunno Intellij suggested that these pattern get factored out as compiled.  for those involved in importing like collada i figured they might somehow contribute to loading speedups